build.sbt

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
```

Comment:
delete license header in .sbt files as it is not required

Reply:
Done